samza-rest/src/test/java/org/apache/samza/monitor/TestLocalStoreMonitor.java

```diff
@@ -78,18 +82,22 @@ public void setUp() throws Exception {
   }
 
   @After
-  public void cleanUp() throws Exception {
+  public void cleanUp()  {
     // clean up the temp files created
-    FileUtils.deleteDirectory(taskStoreDir);
+    try {
+      FileUtils.deleteDirectory(taskStoreDir);
+    } catch (IOException e) {
+      // Happens when task store can't be deleted after test finishes.
+      Assert.fail();
```

Comment:
You can just use a Logger to log in that case. e.getMessage isn't usually sufficient.